config/locales/en/helpers.yml

```diff
@@ -88,6 +88,9 @@ en:
         expert_examination_justification: *ioj_justification_hint
         interest_of_another_justification: *ioj_justification_hint
         other_justification: *ioj_justification_hint
+      steps_evidence_upload_form:
+        #TODO update max file size
```

Comment:
Do we want to have an overall limit as well?

Reply:
I'll put it in the TODO. Had just been following the designs